components/ambassadors/meet/Meet.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC (http://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import * as React from 'react';
+import { Row, Col, Container, Card } from 'react-bootstrap';
+
+import Ambassadors from '../../../_data/ambassadors.json';
+
+import styles from './Meet.module.css';
+
+export default function Meet(props) {
+
+  const ambassadors = Ambassadors.ambassadors;
+
+  return (
+    <Col xs={12}>
+      <Container>
+        <Row>
+          <Col xs={12}>
+            <h2 id='meet-our-ambassadors' className='section'>
```

Comment:
Is this section hidden? We don't have ambassadors yet.

Reply:
Yes